crates/turbo-tasks-memory/src/viz/mod.rs

```diff
@@ -29,14 +29,14 @@ fn get_id<'a>(ty: &'a TaskType, ids: &mut HashMap<&'a TaskType, usize>) -> usize
 }
 
 struct MaxValues {
-    pub total_duration: Duration,
+    pub total_duration: Option<Duration>,
     pub total_current_duration: Duration,
     pub total_update_duration: Duration,
-    pub avg_duration: Duration,
+    pub avg_duration: Option<Duration>,
     pub max_duration: Duration,
     pub count: usize,
     pub active_count: usize,
-    pub updates: usize,
+    pub updates: Option<usize>,
```

Comment:
These three values can't be computed without full stats, but this struct has no concept of full vs. not-full stats. I thought using an Option was the clearer option instead of defaulting to a 0-value, since we might want to handle the two cases separately. But I agree it makes the option-handling code more complex.